src/setting.c

```diff
@@ -693,6 +696,32 @@ static int user_style(Client *c, const char *name, DataType type, void *value, v
     return CMD_SUCCESS;
 }
 
+static int style_sheet_file(Client *c, const char *name, DataType type, void *value, void *data)
+{
+    char *path;
+    gchar *source;
+
+    path = util_get_config_dir();
+
+    vb.files[FILES_USER_STYLE] = util_get_filepath(path, value, FALSE);
+
+    WebKitUserContentManager *ucm;
+    ucm = webkit_web_view_get_user_content_manager(c->webview);
+
+    WebKitUserStyleSheet *style;
+    g_file_get_contents(vb.files[FILES_USER_STYLE], &source, NULL, NULL);
+    style = webkit_user_style_sheet_new(
+        source, WEBKIT_USER_CONTENT_INJECT_ALL_FRAMES,
+        WEBKIT_USER_STYLE_LEVEL_USER, NULL, NULL
+    );
+
+    webkit_user_content_manager_add_style_sheet(ucm, style);
```

Comment:
I think you have to remove previouly set style sheets. Else the styles will be merged together internally.